src/middlewared/middlewared/plugins/vm/pci.py

```diff
@@ -70,109 +143,22 @@ def retrieve_node_information(self, xml):
         Bool('reset_mechanism_defined', required=True),
         Str('error', null=True, required=True),
         Str('device_path', null=True, required=True),
+        Str('description', empty=True, required=True),
         register=True,
     ))
-    async def passthrough_device(self, device):
```

Comment:
Can we add `controller_type` above to `returns` schema please?
Also with the new changes `controller_type` can be `null` - this is for my case where the old implementation reported `Unassigned class` - i believe it can be `null` and we should just reflect that properly?
```
{
  "capability": {
    "class": "0x130000",
    "domain": "0",
    "bus": "46",
    "slot": "0",
    "function": "0",
    "product": "Starship/Matisse PCIe Dummy Function",
    "vendor": "Advanced Micro Devices, Inc. [AMD]"
  },
  "iommu_group": {
    "number": 30,
    "addresses": [
      {
        "domain": "0x0000",
        "bus": "0x2e",
        "slot": "0x00",
        "function": "0x0"
      }
    ]
  },
  "controller_type": "Unassigned class",
  "critical": false,
  "device_path": "/sys/bus/pci/devices/0000:2e:00.0",
  "drivers": [],
  "available": true,
  "error": null,
  "reset_mechanism_defined": true,
  "description": "0000:2e:00.0 'Unassigned class': Starship/Matisse PCIe Dummy Function by 'Advanced Micro Devices, Inc. [AMD]'"
}
```

This is from the older implementation ^^^